### DISCUSSION TITLE
No microphone support passthrough in host to client (like it's implemented with audio sound output)

### DISCUSSION THREAD

**Discussion by ushabuntu** [rustdesk/rustdesk#6933] [General]
When connected to the host device - playing audio in host works fine and plays audio in connected client
This is not the case with microphone, and it's very useful - e.g. when connecting to client and using VOIP or Teams / Zoom calls
The feature - when working as intended - will allow client to use microphone and passthrough the mic output to the host machine (as in sound / audio input)

**Comment by Misenkooo** [comment]
Still nothing ? Can't believe this is not supported.

**Comment by Arrmm** [comment]
I second for this request.  It is a much needed functionality that shouldn't be too difficult to implement.

**Comment by playswithfir3** [comment]
would like to bump this request, would be quite useful for gaming

**Reply by Arrmm** [reply]
How do you game on Rustdesk?  Unless it is CRPG or non-action game otherwise the refresh rate won't be quick enough.  Unless there have done something close to Parsec?

**Comment by stefanleh** [comment]
bump

**Comment by Victorious70** [comment]
bump

**Comment by hubert17** [comment]
bump

**Comment by Sharkfood1375** [comment]
bump

**Comment by medeiroshudson** [comment]
bump

**Comment by jrab66** [comment]
same question from my side.

**Comment by VasGuruDev** [comment]
Bump

**Comment by WahidBitar** [comment]
Bump
